src/test/tor/minimal/tor-minimal.yaml

```diff
@@ -100,6 +100,19 @@ hosts:
             --defaults-torrc torrc-defaults -f torrc
       start_time: 60
       expected_final_state: running
+  ptbridge:
+    network_node_id: 0
+    ip_addr: 100.0.0.100
+    processes:
+    - path: tor
+      args: --Address ptbridge --Nickname ptbridge
+            --BridgeRelay 1
+            --ServerTransportPlugin "obfs4 exec /usr/bin/obfs4proxy --enableLogging --logLevel=DEBUG --unsafeLogging"
```

Comment:
Will tor do $PATH-based lookups? If so, it would be nice to not hardcode a obfs4proxy path.

Reply:
It won't. Even if it did, we don't set the `PATH`, or provide an easy way to propagate the user's `PATH` into managed processes.

We have the same issue with the `python3` in the fork+exec tests.

I'm still thinking about the best way to address it. For our tests the best solution I can think of that would be robust to odd layouts (e.g. nix, guix) would be to use CMake to locate binaries using the build-user's PATH, and then patch the fully qualified path into generated configs. It's a bit gross though :/.